wcomponents-core/pom.xml

```diff
@@ -272,6 +272,24 @@
 			<artifactId>tika-core</artifactId>
 			<version>1.22</version>
 		</dependency>
+	 <dependency>
+	  <groupId>org.seleniumhq.selenium</groupId>
+	  <artifactId>selenium-java</artifactId>
+	  <scope>test</scope>
+	  <version>2.44.0</version>
+	 </dependency>
+	 <dependency>
+	  <groupId>com.opera</groupId>
+	  <artifactId>operadriver</artifactId>
+	  <scope>test</scope>
+	  <version>1.5</version>
+	  <exclusions>
+	   <exclusion>
+	    <groupId>org.seleniumhq.selenium</groupId>
+	    <artifactId>selenium-remote-driver</artifactId>
+	   </exclusion>
+	  </exclusions>
+	 </dependency>
 	</dependencies>
```

Comment:
Not sure why these selenium dependencies have been added (also the indenting is a mix of space and tab)

Reply:
Weird. I'm not sure why this is here either. I'll remove it.